lib/routes/buct/gr.ts

```diff
@@ -0,0 +1,90 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/gr/:type',
+    categories: ['university'],
+    example: '/buct/gr/jzml',
+    parameters: {
+        type: {
+            type: 'string',
```

Comment:
This does not match with the [type definition](https://github.com/DIYgod/RSSHub/blob/abc1a579a45c8e9c2a14de8abb557195b90e119d/lib/types.ts#L293).